segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperLogFactory.java

```diff
@@ -148,11 +147,14 @@ private BookKeeper startBookKeeperClient() throws Exception {
             config.setTLSTrustStorePasswordPath(this.config.getTlsTrustStorePasswordPath());
         }
 
+        String metadataServiceUri = "zk://" + this.config.getZkAddress();
```

Comment:
this is not enough if you have multiple zookeeper servers, you have to replace ',' with ';'
see here:
https://github.com/apache/bookkeeper/blob/4c8587715104cde6dbd70d1e7de0fc1853122eda/bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java#L259

Reply:
Done.